UnityProject/Assets/Scripts/HealthV2/Living/BodyPartModification.cs

```diff
@@ -9,14 +9,23 @@ public class BodyPartModification : MonoBehaviour
 	{
 		[NonSerialized]
 		public BodyPart RelatedPart;
-		public virtual void ImplantPeriodicUpdate() { }
+		public virtual void ImplantPeriodicUpdate() 
+		{ 
+			if(RelatedPart.IsBleedingInternally)
+			{
+				InternalDamageLogic();
```

Comment:
I'd like to just say that I had a conversation with someone on discord about this at some point and they told me it made more sense to offload that logic into BodyPartModification rather than replacing the entire BodyPart class with virtual functions.
And they were right, this is a cleaner approach and makes much more sense because we're dealing with custom body part code inside of BodyPartModification.
So I'm leaving it this way.

Reply:
Don't resolve conversations. We are the ones that mark them as resolved. 

Please explain to me why is it cleaner to make a class check the state of another class just to call a method that will tell the second class to do something.